src/server/hset_family.cc

```diff
@@ -215,15 +257,27 @@ OpStatus OpIncrBy(const OpArgs& op_args, string_view key, string_view field, Inc
   return OpStatus::OK;
 }
 
-OpResult<StringVec> OpScan(const OpArgs& op_args, std::string_view key, uint64_t* cursor) {
+OpResult<StringVec> OpScan(const OpArgs& op_args, std::string_view key, uint64_t* cursor,
+                           const ScanOpts& scan_op) {
+  using PrivateDataRef = std::tuple<StringVec&, const ScanOpts&>;
+  static constexpr const size_t HASH_TABLE_ENTRIES_FACTOR = 2;  // return key/value
```

Comment:
no need `static constexpr const`. `constexpr` is enough. here and everywhere

Reply:
done